UM/Mesh/ReadMeshJob.py

```diff
@@ -68,15 +68,19 @@ def run(self):
         if hasattr(Application.getInstance().getController().getScene(), "_maximum_bounds"):
             max_bounds = Application.getInstance().getController().getScene()._maximum_bounds
             node._resetAABB()
-            bounding_box = node.getBoundingBox()
+            build_bounds = node.getBoundingBox()
 
             if Preferences.getInstance().getValue("mesh/scale_to_fit") == True or Preferences.getInstance().getValue("mesh/scale_tiny_meshes") == True:
-                scale_factor_width = max_bounds.width / bounding_box.width
-                scale_factor_height = max_bounds.height / bounding_box.height
-                scale_factor_depth = max_bounds.depth / bounding_box.depth
-                scale_factor = min(scale_factor_width,scale_factor_height,scale_factor_depth)
-                if Preferences.getInstance().getValue("mesh/scale_to_fit") == True and (scale_factor_width < 1 or scale_factor_height < 1 or scale_factor_depth < 1):
-                    # Use scale factor to scale large object down
+                scale_factor_width = max_bounds.width / build_bounds.width
+                scale_factor_height = max_bounds.height / build_bounds.height
+                scale_factor_depth = max_bounds.depth / build_bounds.depth
+                scale_factor = min(scale_factor_width, scale_factor_depth, scale_factor_height)
+                if Preferences.getInstance().getValue("mesh/scale_to_fit") == True and (scale_factor_width < 1 or scale_factor_height < 1 or scale_factor_depth < 1): # Use scale factor to scale large object down
+                    # Ignore scaling on models which are less than 3 times bigger than the build volume
```

Comment:
Sorry for the late reply. Was learning for exams with a friend. 
Do I have to rework this and the following as the PR has been already merged?